pkg/registry/client.go

```diff
@@ -131,6 +135,94 @@ func NewClient(options ...ClientOption) (*Client, error) {
 	return client, nil
 }
 
+func NewCrosClient(chartref string, options ...ClientOption) (*Client, error) {
```

Comment:
what is a "Cros" client?

Reply:
**Cros  (Connection Roundtripper Over Ssl)** 
This **NewCrosclient** only uses two-way TLS support. Due to less experience in golang development, I thought maybe my code affect existing functionality so I created a new function named as newCrosClient.

If that naming convention does not match helm expectations we can change that.